scripts/validate_docstrings.py

```diff
@@ -497,6 +497,44 @@ def parameter_desc(self, param):
                 desc = desc[: desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        # The 'Parameters' value of NumpyDocString._parsed_data is a list of
+        # NamedTuples.
+        parsed_param = [
+            p for p in self.doc._parsed_data["Parameters"] if p.name == param
+        ][0]
```

Comment:
Is there not a better way to access the required attribute here than by the initial index? Not overly familiar with the data structure but this is not super readable at first glance